yarn.lock

```diff
@@ -5386,9 +5411,83 @@
   resolved "https://registry.yarnpkg.com/@spectrum-css/vars/-/vars-9.0.8.tgz#6af3bcdace903b8461f5fcd4c9aa23e70128a456"
   integrity sha512-rGfd7jqXOdR69bEjrRP58ynuIeJU0czPfwQvzhtCzg7jKVukV+efNHqrs086sC6xutB3W4TF71K/dZMr3oyTyg==
 
+"@spectrum-web-components/base@^0.40.5":
```

Comment:
Why these changes?

Reply:
These should be transitive depedencies. If you run a yarn install on `main` branch you can see these changes

Reply:
I don't think they belong. Do you have a `react` directory locally?